bitnami/tomcat/values.yaml

```diff
@@ -103,9 +103,15 @@ tomcatUsername: user
 ## ref: https://github.com/bitnami/containers/tree/main/bitnami/tomcat#creating-a-custom-user
 ##
 tomcatPassword: ""
-## @param existingSecret Use existing secret for password details (`tomcatPassword` will be ignored and picked up from this secret). The secret has to contain the key `tomcat-password`
+## @param existingSecret Use existing secret for credentials details (`tomcatPassword` will be ignored and picked up from this secret. `tomcatUsername` might be ignored depending of provided `adminUsernameKey` value).
 ##
 existingSecret: ""
+## @param secretKeys.adminUsernameKey Name of key in existing secret to use for Tomcat admin username (overrides `tomcatUsername` if not empty). Only used when `existingSecret` is set.
+## @param secretKeys.adminPasswordKey Name of key in existing secret to use for Tomcat admin password (overrides `tomcatPassword`). Only used when `existingSecret` is set.
+##
+secretKeys:
+  adminUsernameKey: ""
+  adminPasswordKey: "tomcat-password"
```

Comment:
You could use an empty value since the logic above defaults to `tomcat-password`.